### PR TITLE
PixelPaint: Propagate errors from making tool property widgets

### PR DESCRIPTION
**Pull request by krkk**

### CODE REVIEW

File: Userland/Applications/PixelPaint/Tools/EraseTool.cpp [hunk 1]
```diff
@@ -53,84 +53,85 @@ void EraseTool::draw_point(Gfx::Bitmap& bitmap, Gfx::Color color, Gfx::IntPoint
...
 }
 
-GUI::Widget* EraseTool::get_properties_widget()
+ErrorOr<GUI::Widget*> EraseTool::get_properties_widget()
 {
     if (!m_properties_widget) {
-        m_properties_widget = GUI::Widget::construct();
-        m_properties_widget->set_layout<GUI::VerticalBoxLayout>();
+        auto properties_widget = TRY(GUI::Widget::try_create());
+        (void)TRY(properties_widget->try_set_layout<GUI::VerticalBoxLayout>());
```
Comment (konradekk) [inline Userland/Applications/PixelPaint/Tools/EraseTool.cpp:60]: (suspicious api… it seems we never use the return value for this method here. which is somewhat odd. 🧐)

**Review by AtkinsSJ — APPROVED** [review]
Looks good to me! You just need to fix up the merge conflict.

File: Userland/Applications/PixelPaint/Tools/EraseTool.cpp (hunk 1 above)
Comment (krkk) [reply Userland/Applications/PixelPaint/Tools/EraseTool.cpp:60]: iiiii- don’t think so honestly. we are not setting any layout properties like spacing or margin so there’s no need to assign it to a variable.

Comment (AtkinsSJ) [reply Userland/Applications/PixelPaint/Tools/EraseTool.cpp:60]: I've just self-yakbaited myself into changing the Layout API. :laughing:

**Review by AtkinsSJ — APPROVED** [review]

**Pull request merged**